### DISCUSSION TITLE
Researching into 4mmc + Alcohol

### DISCUSSION THREAD

**Discussion by utaninja** [TripSit/drugs#18] [Drug Interactions]
This is just to start a log of what I've collected research wise in regards to alcohol + 4mmc
https://pubmed.ncbi.nlm.nih.gov/26747301/
https://pubmed.ncbi.nlm.nih.gov/27562197/
https://pubmed.ncbi.nlm.nih.gov/32063845/
https://pubmed.ncbi.nlm.nih.gov/29349706/
https://pubmed.ncbi.nlm.nih.gov/26228024/

Reading through trip reports, there doesn't seem anything to be of note that I found worth adding. Most trip reports including alcohol also included multiple other drugs.

I will continue to update this! Open to input always!

**Comment by utaninja** [comment]
I am leaning to putting this as caution. Typical stimulant+alcohol interaction warning- in addition to noting the combination increasing side effects / strain onto the body/mind.

**Reply by jedrek8733** [reply]
I agree with "caution"

**Comment by utaninja** [comment]
From @zelixir25
"Seems similar to alcohol + other stimulants/amphetamines. Alcohol can decrease some of the stimulation without inhibiting the euphoria and the 4mmc reduces the intoxicating effects of alcohol. Some studies indicate that this can cause people to use more of both than they normally would. Unsure of how to classify this. Caution? Safe?

Berquist, M. D. I., Peet, M. M., & Baker, L. E. (2015). Behavioral sensitization following concurrent exposure to mephedrone and D-amphetamine in female mice. Behavioural Pharmacology, 26(1 and 2-Special Issue), 180. https://doi.org/10.1097/FBP.0000000000000121

Farré, M., Perez-Maña, C., Souza, E. de, Mateus, J., Theunisen, E., Kuypers, K., Ramaekers, J., Fonseca, F., Torrens, M., Olesti, E., Torre, R. de la, & Papaseit, E. (2016). Interactions between mephedrone and alcohol in humans: Cardiovascular and subjective effects. European Psychiatry, 33(S1), S115–S115. https://doi.org/10.1016/j.eurpsy.2016.01.122

Pieprzyca, E., Skowronek, R., & Czekaj, P. (2022). Toxicological Analysis of Intoxications with Synthetic Cathinones. Journal of Analytical Toxicology, 46(7), 705–711. https://doi.org/10.1093/jat/bkab102"

**Comment by zelixir25** [comment]
According to the paper below:
> Further studies confirmed that mephedrone limits alcohol-caused sedation and feeling drunk and showed that co-administration of both drugs caused more severe cardiovascular effects and more pronounced feeling of euphoria than both substances taken separately. Moreover, it seems that the tendency to abuse mephedrone used concurrent with alcohol is higher than that caused by mephedrone alone [92,94]. Co-administration of mephedrone and alcohol also increases the amounts of alcohol consumed [95]. Taken together, combining mephedrone and alcohol poses a higher risk of intoxication and dependence development [92,94,95].

Bielecka-Papierz, G., Poleszak, E., & Serafko, A. (2023). Mephedrone–a synthetic derivative of cathinone. Current Issues in Pharmacy and Medical Sciences, 36(1), 54-64.

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/issues/308
